### PR TITLE
FAQ Edits

### PR DESCRIPTION
**Pull request by micros24**
Tweaked some formatting and added a new FAQ "Steam's file picker no worky".

### CODE REVIEW

File: pages/FAQ/_index.md [hunk 1]
```diff
@@ -328,3 +332,7 @@ If the pop-up disappears immediately, you can use:
 ```ini
 windowrulev2 = minsize 1 1, title:^(TITLE)$, class:^(CLASS)$
 ```
+
+# Steam's file picker no worky
+
+On instances where you have a steam library on another drive that you have to add, the Hyprland's file picker would not normally appear when selecting a directory from steam. Steam have its own file picker, however, it's not functional. Install `xdg-desktop-portal-gtk` to show the desktop's file picker.
```
Comment (vaxerski) [inline pages/FAQ/_index.md:338]: Steam _has_ its own file picker

Comment (micros24) [reply pages/FAQ/_index.md:338]: Yes, steam does have its own file picker. However, their file picker does not work. The file picker window opens but none of the buttons work and you can't select files. The image below is without xdg-desktop-portal-gtk.
swappy-20240212_005252

Comment (o0nd7ots) [reply pages/FAQ/_index.md:338]: I suggest you split the message into multiple lines.

Comment (vaxerski) [reply pages/FAQ/_index.md]: my problem was with the grammar not the content
`have` -> `has`

Comment (micros24) [reply pages/FAQ/_index.md]: Alright, I have made additional commits.

**Review by vaxerski — APPROVED** [review]

**Pull request merged**